internal/e2e/go.mod

```diff
@@ -1,9 +1,9 @@
-module github.com/http-wasm/http-wasm-guest-tinygo/internal
+module github.com/http-wasm/http-wasm-guest-tinygo/e2e
```

Comment:
@codefromthecrypt FYI this was needed to allow individual tests to run in GoLand